src/StreamJsonRpc/SystemTextJsonFormatter.cs

```diff
@@ -1203,34 +1200,55 @@ public object Convert(object value, TypeCode typeCode)
         }
     }
 
+    /// <summary>
+    /// Adds compatibility with DataContractSerializer attributes.
+    /// </summary>
+    /// <example>
+    /// To enable this resolver, add the following when creating your <see cref="SystemTextJsonFormatter"/> instance:
+    /// <code><![CDATA[
+    /// var formatter = new SystemTextJsonFormatter
+    /// {
+    ///     JsonSerializerOptions =
+    ///     {
+    ///         TypeInfoResolver = new SystemTextJsonFormatter.DataContractResolver(),
+    ///     },
+    /// };
+    /// ]]></code>
+    /// </example>
     private class DataContractResolver : IJsonTypeInfoResolver
```

Comment:
Shouldn't this be public for users to be able to use it?

Reply:
Per the PR description, I'm ripping it out.